core/embed/rust/src/ui/component/text/layout.rs

```diff
@@ -264,7 +264,15 @@ impl TextLayout {
         {
             sink.prev_page_ellipsis(*cursor, self);
             // Move the cursor to the right, always the same distance
-            cursor.x += self.style.prev_page_ellipsis_width();
+            // Special case in chunkifying text - move the cursor so that we
+            // start with the second chunk.
+            if let Some(chunk_config) = self.style.chunks {
+                // Assuming mono-font, so all the letters have the same width
```

Comment:
That is not right, right? Regarding **w** and **m**.

Reply:
It does not matter here, we just want to offset the first chunk and the space (which combined should always be constant). 

(m, M, w and W are accounted for elsewhere, we are decreasing the space between chunks when we encounter them)

Reply:
Wouldn't it change with [this](https://github.com/trezor/trezor-firmware/issues/3272)?

Reply:
I am little confused with this second issue you mention here - what is the goal there?